lib/lumberg/whm/transfer_tool.rb

```diff
@@ -0,0 +1,112 @@
+module Lumberg
+  module Whm
+    class TransferTool < Base
+
+      # Public: Creates a new transfer session.
+      #
+      # options - Hash options for API call params. (default: {})
+      #   :host              - String hostname of the remote server.
+      #   :user              - String username to connect on remote server.
+      #   :password          - String password for account specified above.
+      #
+      # Returns Hash API Response.
+      def create(options = {})
+        default_options = { remote_server_type: 'auto',
+                            port: 22,
+                            root_escalation_method: 'sudo',
+                            transfer_threads: 2,
+                            restore_threads: 2,
+                            unrestricted_restore: 1,
+                            compressed: 1,
+                            unencrypted: 0,
+                            use_backups: 1,
+                            low_priority: 0,
+                            enable_custom_pkgacct: 0 }
+
+        options.merge!({ "api.version" => "1" })
```

Comment:
Can you remove the curly braces here and in the other `merge!` calls below? We typically skip them in method calls like this.